tools/qa/buildkite_build_horton_inplace.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+source tools/qa/buildkite_common.sh
+get_ancestor  # Writes $ANCESTOR_SHA variable.
+
+echo "--- Basic source tests"
+./tools/qa/check_names.py
+
+echo "--- Build Cython files & HORTON"
+./cleanfiles.sh
+CC="ccache gcc" ./setup.py build_ext -i
```

Comment:
Hmm. There's another script that runs before it to checkout the merge? It's called in buildkite, but maybe it's not working?